### ISSUE TITLE
Project does not run

### ISSUE THREAD

**Issue by MelvinG24** [tetreum/carmechanic#49]
I was trying to open this project on unity, but it's look like some files or plugin are mission. I was trying to open this project in different Unity's version but without any luck. 

One of the most common error I have while opening this project is with the Sercer.cs file. It's said that service mission two arguments which I don't know is one are.

**Comment by fgsoftware1** [comment]
> I was trying to open this project on unity, but it's look like some files or plugin are mission. I was trying to open this project in different Unity's version but without any luck.
> 
> One of the most common error I have while opening this project is with the Sercer.cs file. It's said that service mission two arguments which I don't know is one are.

probably the problem would be plugins that don't have newr unity versions support. i0ll take a look later today and make update you about the issue :)

**Comment by tetreum** [comment]
I'm afraid the project is not actively maintained. I've upgraded it to the latest Unity LTS but i've tried the gameplay and it's broken, so it still needs fixes. 
@fgsoftware1 Havok should be removed as no Pro/Enterprise features should be used.

**Comment by fgsoftware1** [comment]
> I'm afraid the project is not actively maintained. I've upgraded it to the latest Unity LTS but i've tried the gameplay and it's broken, so it still needs fixes. @fgsoftware1 Havok should be removed as no Pro/Enterprise features should be used.

yeah i know, i don't even know why this DOTS stuff is there. i'll try migrate the proj to latest version and see what else i can do to try make it playable again

**Comment by tetreum** [comment]
@fgsoftware1 i've checked why nothing on the gameplay is working:
- The engine is missing `CarEngine` component. If added (to the car or the engine), the car can be disassembled again. 
- The engine doesn't seem to have any collider, that's why it can't be interacted. I believe each part used to have a collider + CarPart script.
- The interface that told you in which mode you were is no longer present
- Outline shader isn't working properly either:
![image](https://github.com/tetreum/carmechanic/assets/1708730/bda3a172-8660-47c7-8f8d-e08cb0833c8a)

**Comment by fgsoftware1** [comment]
> Reference in

i think you can generate collider on 3d obj properties

**Comment by fgsoftware1** [comment]
> @fgsoftware1 i've checked why nothing on the gameplay is working:
> 
> * The engine is missing `CarEngine` component. If added (to the car or the engine), the car can be disassembled again.
> * The engine doesn't seem to have any collider, that's why it can't be interacted. I believe each part used to have a collider + CarPart script.
> * The interface that told you in which mode you were is no longer present
> * Outline shader isn't working properly either:
>   ![image](https://user-images.githubusercontent.com/1708730/255389595-bda3a172-8660-47c7-8f8d-e08cb0833c8a.png)

i think you can generate collider on 3d obj properties

**Comment by fgsoftware1** [comment]
> > @fgsoftware1 i've checked why nothing on the gameplay is working:
> > 
> > * The engine is missing `CarEngine` component. If added (to the car or the engine), the car can be disassembled again.
> > * The engine doesn't seem to have any collider, that's why it can't be interacted. I believe each part used to have a collider + CarPart script.
> > * The interface that told you in which mode you were is no longer present
> > * Outline shader isn't working properly either:
> >   ![image](https://user-images.githubusercontent.com/1708730/255389595-bda3a172-8660-47c7-8f8d-e08cb0833c8a.png)
> 
> i think you can generate collider on 3d obj properties

ant the script isn't created or there are issues with it

**Comment by tetreum** [comment]
The script is fine:

https://github.com/tetreum/carmechanic/assets/1708730/446624fe-9ca6-45f8-a6a9-196f1b22ccdc

**Comment by fgsoftware1** [comment]
> > > @fgsoftware1 i've checked why nothing on the gameplay is working:
> > > 
> > > * The engine is missing `CarEngine` component. If added (to the car or the engine), the car can be disassembled again.
> > > * The engine doesn't seem to have any collider, that's why it can't be interacted. I believe each part used to have a collider + CarPart script.
> > > * The interface that told you in which mode you were is no longer present
> > > * Outline shader isn't working properly either:
> > >   ![image](https://user-images.githubusercontent.com/1708730/255389595-bda3a172-8660-47c7-8f8d-e08cb0833c8a.png)
> > 
> > 
> > i think you can generate collider on 3d obj properties
> 
> ant the script isn't created or there are issues with it

why the script is white then?
